ios/browser/api/ads/brave_ads.mm

```diff
@@ -1145,6 +1145,17 @@ - (void)loadAdsResource:(const std::string&)id
   callback(/* success */ false, "");
 }
 
+- (void)clearScheduledCaptcha {
+}
+
+- (void)getScheduledCaptcha:(const std::string&)payment_id
+                   callback:(ads::GetScheduledCaptchaCallback)callback {
+}
+
+- (void)showScheduledCaptchaNotification:(const std::string&)payment_id
+                               captchaId:(const std::string&)captcha_id {
+}
```

Comment:
Can we add some comments to these functions as to why they're empty? (Something simple like "iOS doesn't support adaptive captcha so these methods won't get called/don't need to be handled")

Reply:
Good point, added.